glib/glib.go

```diff
@@ -371,7 +371,7 @@ func idleAdd(priority Priority, f interface{}) SourceHandle {
 // After running once, the source func will be removed from the main event loop,
 // unless f returns a single bool true.
 func TimeoutAdd(milliseconds uint, f interface{}) SourceHandle {
-	return timeoutAdd(milliseconds, false, PRIORITY_DEFAULT, f)
+	return timeoutAdd(milliseconds, false, PRIORITY_DEFAULT_IDLE, f)
```

Comment:
I think for the timeouts PRIORITY_DEFAULT is correct, only the idle function needs PRIORITY_DEFAULT_IDLE,
see https://developer.gnome.org/glib/stable/glib-The-Main-Event-Loop.html#g-idle-add and https://developer.gnome.org/glib/stable/glib-The-Main-Event-Loop.html#g-timeout-add

Reply:
This is now fixed.